presto-hive/src/main/java/com/facebook/presto/hive/util/ConfigurationUtils.java

```diff
@@ -61,4 +70,38 @@ public static JobConf toJobConf(Configuration conf)
         }
         return new JobConf(conf);
     }
+
+    public static JobConf configureCompression(Configuration config, HiveCompressionCodec compression)
+    {
+        JobConf result = new JobConf(false);
+        copy(config, result);
```

Comment:
Curious, what's the difference between doing this vs. https://github.com/prestodb/presto/blob/6727da11e22c9682c21c409adda1a3fc4680c903/presto-hive/src/test/java/com/facebook/presto/hive/benchmark/FileFormat.java#L421

Reply:
> JobConf config = new JobConf(conf); 

This will re-read the configuration xml files in the classpath  :smile: